osgp/protocol-adapter-oslp/osgp-adapter-protocol-oslp-elster/src/main/java/org/opensmartgridplatform/adapter/protocol/oslp/elster/domain/entities/OslpDevice.java

```diff
@@ -60,7 +57,7 @@
     private final SecureRandom random = new SecureRandom();
 
     @Transient
-    private Integer sequenceNumberMaximum = 65535;
+    private final Integer sequenceNumberMaximum = 65535;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1170)